src/lib/webtab/webtab.cpp

```diff
@@ -292,6 +295,26 @@ void WebTab::setPinned(bool state)
     m_isPinned = state;
 }
 
+bool WebTab::isMuted() const
```

Comment:
One small thing, static analyzer issues about reaching the end of non-void function, when you are building with Qt <5.7.0
I assume, it's not a big deal, since eventually all code will be adjusted to newer versions of Qt?